instrumentation-api/src/test/java/io/opentelemetry/instrumentation/api/internal/OperationMetricsUtilTest.java

```diff
@@ -5,6 +5,8 @@
 
 package io.opentelemetry.instrumentation.api.internal;
 
+import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.assertThat;
```

Comment:
looks like no need for OpenTelemetryAssertions in this class
```suggestion
static import org.assertj.core.api.Assertions.assertThat;
```